Frida/Script.swift

```diff
@@ -214,4 +251,55 @@ public class Script: NSObject, NSCopying {
     private let releaseConnection: GClosureNotify = { data, _ in
         Unmanaged<SignalConnection<Script>>.fromOpaque(data!).release()
     }
+    
+    // MARK: - RPC Types
+    
+    @dynamicMemberLookup
+    public struct Exports {
+        unowned var script: Script?
+        
+        init(script: Script) {
+            self.script = script
+        }
+        
+        subscript(dynamicMember functionName: String) -> RpcFunction {
+            get {
+                return RpcFunction(script: self.script, functionName: functionName)
+            }
+        }
+    }
+    
+    // MARK: - RPC
+    
+    public lazy var exports: Exports = {
+        return Exports(script: self)
+    }()
```

Comment:
Would be good to move this property up to right after `deinit`. (The style used throughout Frida is "newspaper ordering" as per [Clean Code](https://www.amazon.com/dp/0132350882/) chapter 5, which roughly quoted goes something like: `Reading code should work kind of like reading a newspaper article: the high-level details come first (at the top) and you get more detail as you read down. If one function calls another, they should be vertically close, and the caller should be above the callee, if at all possible.`)

Reply:
Appreciate the thoughts behind this. I love that book.